README.md

```diff
@@ -2302,6 +2303,34 @@ Server: spray-can/1.3.3
 
 ```
 
+
+## GET /api/workflows/:version/backends
+
+This endpoint returns a list of the backends supported by the server.
+
+cURL:
+```
+$ curl http://localhost:8000/api/workflows/v1/backends
+```
+
+HTTPie:
+```
+$ http http://localhost:8000/api/workflows/v1/backends
+```
+
+Response:
+```
+HTTP/1.1 200 OK
```

Comment:
Spoke in person but this was just me floating a trial balloon to see if anyone really wanted this stuff around. I'll probably open a PR at some point when i'm poking at the readme again